package/kedro_viz/server.py

```diff
@@ -198,9 +199,16 @@ def pretty_name(name):
     edges = []
     namespace_tags = defaultdict(set)
     all_tags = set()
+    namespace_to_layer = {}
+
+    data_set_to_layer_map = {
```

Comment:
Minor minor suggestion. `namespace_to_layer` is also a dict and the `a_to_b` usually clearly suggests a dictionary, so `map` seems superfluous. Plus you a get a shorted name! :D
```suggestion
    dataset_to_layer = {
```

Reply:
Any reason why this was resolved?